shell/common/isolate_configuration.cc

```diff
@@ -153,6 +154,10 @@ std::unique_ptr<IsolateConfiguration> IsolateConfiguration::InferFromSettings(
     return nullptr;
   }
 
+  if (settings.application_kernels) {
+    return CreateForKernelList(settings.application_kernels());
+  }
+
```

Comment:
Add a TODO to kill off the following conditional block

Reply:
I have noted that the fields in the settings are deprecated. Once those are removed, this check will necessarily have to go as well.